jiant/config/wmt14_en_de.conf

```diff
@@ -0,0 +1,51 @@
+// Template for translation as a seq2seq problem.
```

Comment:
Say a bit more about where these values come from. Are they supposed to match a paper? Which ones are necessary parts of the setup, and which are just things that you tuned?

Reply:
Mention that this is a non-Transformers setup, and say what it'd take to use a Transformers model.

Reply:
Done.